tests/suites/test_suite_pem.function

```diff
@@ -3,6 +3,8 @@
 #include "mbedtls/pem.h"
 #include "mbedtls/des.h"
 #include "mbedtls/aes.h"
+#include "legacy_or_psa.h"
```

Comment:
```suggestion
#if defined(MBEDTLS_USE_PSA_CRYPTO)
#include "psa/crypto.h"
#endif
#include "legacy_or_psa.h"
```

Reply:
The test framework automatically includes `psa/crypto.h`. It should probably include `legacy_or_psa.h` automatically as well.

Reply:
Also, `legacy_or_psa.h` should include `psa/crypto.h`. Right now, including `legacy_or_psa.h` doesn't work if `psa/crypto.h` hasn't already been included.

Reply:
I think is is good idea to include `psa/crypto.h` in `legacy_or_psa.h`.